go/libkb/secret_store.go

```diff
@@ -90,12 +90,14 @@ func NewSecretStore(g *GlobalContext, username NormalizedUsername) SecretStore {
 }
 
 func GetConfiguredAccounts(c SecretStoreContext, s SecretStoreAll) ([]keybase1.ConfiguredAccount, error) {
-	currentUsername, otherUsernames, err := c.GetAllUserNames()
+	currentUsername, allUsernames, err := c.GetAllUserNames()
 	if err != nil {
 		return nil, err
 	}
 
-	allUsernames := append(otherUsernames, currentUsername)
+	if currentUsername != "" {
+		allUsernames = append(allUsernames, currentUsername)
+	}
```

Comment:
Do we wanna filter on other ones from `allUsernames` too or does it never contain empty strings?